gobblin-cluster/src/main/java/org/apache/gobblin/cluster/HelixUtils.java

```diff
@@ -278,13 +278,16 @@ static void waitJobCompletion(HelixManager helixManager, String workFlowName, St
           case STOPPING:
             log.info("Waiting for job {} to complete... State - {}", jobName, jobState);
             Thread.sleep(TimeUnit.SECONDS.toMillis(1L));
+            if (stoppingStateEndTime == null) {
+              stoppingStateEndTime = jobStartTimeMillis + stoppingStateTimeoutInSeconds * 1000;
```

Comment:
why start from jobStartTimeMills? Shouldn't it be currentTimeStamp? Basically it should be the first time we see job in stopping state plus the stopping state timeout.

Reply:
Also add one log here indicate we see job in stopping state so that you will be able to test whether your fix works?

Reply:
+1. Current impl preserves the bug we are trying to fix. Stop time starts from
System current millis

Reply:
Done. 
Log can be traced by line 279 if the job is STOPPING state.